src/core/net/srp_client.cpp

```diff
@@ -2080,6 +2096,9 @@ void Client::ProcessAutoStart(void)
 
     if (IsRunning())
     {
+        // Use long delays when the SRP client is already running,
+        // and there is a change in SRP server address
+        mShouldUseLongDelays = true;
```

Comment:
Move this after the `VerifyOrExit((GetServerAddress() != serverSockAddr) || shouldRestart)` statement?

We want this to take effect only when there is a change.

Reply:
thanks for catching this, moved it